### PR TITLE
Split development with Vietnam API

### PR DESCRIPTION
**Pull request by VietThan**
Related: #5

### CODE REVIEW

**Comment by VietThan** [comment]
Used git submodule.

**Comment by VietThan** [comment]
@nguyenkhoa2407 review plz

File: .vscode/.ropeproject/config.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
+# The default ``config.py``
+# flake8: noqa
+
+
+def set_prefs(prefs):
+    """This function is called before opening the project"""
```
Comment (nguyenkhoa2407) [inline .vscode/.ropeproject/config.py]: What does this file do :o

**Comment by VietThan** [comment]
This PR will:
- Split the development of data API with [VietnamAPI](github.com/VietThan/VietnamAPI) using git submodule
- Rewrote homepage js script to use fetch and rely on pulling data from VietnamAPI

The website is in version 1.0 with this PR.

**Review by nguyenkhoa2407 — APPROVED** [review]
looks good!

**Pull request merged**